NEWS.md

```diff
@@ -81,6 +81,8 @@ unit = "s")
 
 14. Added support for `round()` and `trunc()` to extend functionality of `ITime`. `round()` and `trunc()` can be used with argument units: "hours" or "minutes". Thanks to @JensPederM for the suggestion and PR.
 
+15. `DT[,j]` now drops the `"AsIs"` class when `j` returns an object with class `"AsIs"`. Thanks to @ChHaeni for the report and @jangorecki @shrektan for the PR.
```

Comment:
this NEWS item could be expanded, I think the feature is quite useful, a couple use cases are warranted